Formula/cook.rb

```diff
@@ -0,0 +1,34 @@
+class Cook < Formula
+  desc ""
+  homepage ""
+  url "https://github.com/cooklang/CookCLI.git", tag: "v0.0.8"
+  version "0.0.8"
+  license "MIT"
+
+  depends_on xcode: ["10.0", :build]
+
+  patch :DATA
+
+  def install
```

Comment:
Can you add a checksum

e.g. https://github.com/encoredev/homebrew-tap/blob/main/Formula/encore.rb#L7

Reply:
I don't think you can because it's coming from a git repo, not a random file downloaded from the internet.

Reply:
Can it not use the asset from the release? 

https://github.com/cooklang/CookCLI/releases/tag/v0.0.9

Reply:
That doesn’t include builds for Apple M1 (arm64) processors. That’s why I was suggesting we try to merge the formula into homebrew-core: they’ll build for all those different architectures for us.